types/nova-editor-node/index.d.ts

```diff
@@ -79,14 +88,27 @@ declare interface Clipboard {
 
 /// https://docs.nova.app/api-reference/color/
 
-type ColorFormat = 'rgb' | 'hsl' | 'hsb' | 'p3' | 'hex';
-type ColorComponents = [number, number, number, number];
+declare enum ColorFormat {
+    rgb = 'rgb',
+    hsl = 'hsl',
+    hsb = 'hsb',
+    displayP3 = 'p3',
+}
+
+// type ColorComponents = [number, number, number, number];
```

Comment:
Can you remove this if it's not used anymore? Alternatively, I think you could use `[number, number, number] | [number, number, number, number]` to preserve the intent (it makes sure someone can't construct an invalid color, but I understand why it might be a little heavy handed).

Reply:
Let's leave it in as you suggest in case any extensions are using it.